tests/storage/zfs_ng/test_zfs_sr.py

```diff
@@ -134,3 +134,29 @@ def test_vdi_and_snaps_destroy_intermediate_vdi(self, zfs_sr):
         # tell fixture we destroyed that one
         vdis[2:3] = []
         teardown_vdi_and_snaps_chain(zfs_sr, vdis)
+
+@pytest.fixture(scope='package')
```

Comment:
A package-scoped fixture should be in the Package's `conftest.py`, if you expect to use it in several different test modules. Otherwise, `module` scope is enough.

Reply:
that was an outdated commit marked for rework, now reworked